server/src/main/java/org/opensearch/ingest/ConfigurationUtils.java

```diff
@@ -530,10 +531,11 @@ public static Processor readProcessor(
     ) throws Exception {
         String tag = ConfigurationUtils.readOptionalStringProperty(null, null, config, TAG_KEY);
         String description = ConfigurationUtils.readOptionalStringProperty(null, tag, config, DESCRIPTION_KEY);
+        boolean ignoreFailure = ConfigurationUtils.readBooleanProperty(null, null, config, "ignore_failure", false);
```

Comment:
I don't think this line needed to move out of the `if` block, but it should use the `IGNORE_FAILURE_KEY` constant that you defined.